docs/getting-started/advanced/databases.md

```diff
@@ -59,6 +59,8 @@ services:
 !!! danger ""
     Set strong passwords if the database is exposed to an external network. Never expose your database to the public Internet in this way, for example, if it is running on a cloud server.
 
+For increased security do not allow the "photoprism" user to access the database from every host (`'photoprism'@'%'`) but restrict it to the hosts photoprism is accessing the database from (e.g. `'photoprism'@'192.168.1.2'` or `'photoprism'@'192.168.1.%'` for entire subnet).
```

Comment:
Removing `ports` would be better? It would limit access to the Docker network.

Reply:
> Your hint implies running mysql with docker-compose in the same setup.

Yes a docker-compose setup is shown on the lines before this change.

About using or not using Docker, that may go in a different section since right now it only describes the common installation with a container for the database.

https://github.com/photoprism/photoprism/blob/9f1eee6b1e7e6fadac55bb3b6098714648f81bf8/compose.mariadb.yaml#L16-L17

If `ports` are removed, the access from the host or external network won't work anymore, but Photoprism will work without any issue since it's on the same Docker network.